modules/statics/src/ofc.ts

```diff
@@ -237,3 +237,98 @@ export function tofcerc20(
     })
   );
 }
+
+/**
+ * Factory function for ofc stellar token instances.
+ *
+ * @param name unique identifier of the coin
+ * @param fullName Complete human-readable name of the coin
+ * @param network Network object for this coin
+ * @param decimalPlaces Number of decimal places this coin supports (divisibility exponent)
+ * @param asset Asset which this coin represents. This is the same for both mainnet and testnet variants of a coin.
+ * @param kind Differentiates coins which represent fiat assets from those which represent crypto assets
+ * @param prefix? Optional coin prefix. Defaults to empty string
+ * @param suffix? Optional coin suffix. Defaults to coin name.
+ * @param isToken? Whether or not this account coin is a token of another coin
+ * @param features? Features of this coin. Defaults to the DEFAULT_FEATURES defined in `OfcCoin`
+ * @param primaryKeyCurve The elliptic curve for this chain/token
+ */
+export function ofcStellarToken(
+  name: string,
+  fullName: string,
+  decimalPlaces: number,
+  asset: UnderlyingAsset,
+  kind: CoinKind = CoinKind.CRYPTO,
+  features: CoinFeature[] = OfcCoin.DEFAULT_FEATURES,
+  prefix = '',
+  suffix: string = name.replace(/^ofc/, '').toUpperCase(),
```

Comment:
Hm I am not super familiar with adding tokens that aren't erc20s, so I don't have a lot of useful feedback here. One thing though is I noticed `name.replace(/^ofc/, '')` which is something we want to get rid of. In this case it is fine but I see that it is used all over this ofc file. @drew-bitgo Is this something we need to also remove at some point?

Reply:
We dont use `suffix` for anything today, to my knowledge. This is more of a legacy thing that we have due to the base coin types in the sdk. Since its our team that is doing the conversions from ofc to backing 90% of the time, I dont think its a problem, but I would rather have this just be the name, so we dont fall into that trap (and this way, it may help us find places that are using suffix incorrectly)
```suggestion
  suffix: string = name.toUpperCase(),
```

Reply:
fixed